bazel/glog.bzl

```diff
@@ -99,7 +99,6 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
 
     windows_only_srcs = [
         "src/glog/log_severity.h",
-        "src/windows/config.h",
```

Comment:
@drigz It seems I overlooked this last dependency to `windows/config.h`. This should not be necessary anymore, correct? Unfortunately, Bazel build fails without the Windows header.

Reply:
https://github.com/google/glog/commit/30213bf96b9ade105d149bb010332923f11de920

You might need to fold it into this PR as it requires simultaneous deletion of the file.

Reply:
Great, thank you. I had to additionally define `GLOG_NO_ABBREVIATED_SEVERITIES` for Bazel builds on Windows. I hope my change is acceptable.